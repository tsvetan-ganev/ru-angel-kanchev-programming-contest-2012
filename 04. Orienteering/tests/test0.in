7 3
0401Atanasov m 08:05:25 09:05:37
0203Dobreva w 08:20:02 09:40:25
0501Ivanov m 08:00:25 08:59:35
0102Borisov m 08:10:25 09:35:08
0302Hristova w 08:10:13 10:00:10
0701Ivanov m 08:25:25 10:03:05
0602Todorov m 08:30:05 10:01:00