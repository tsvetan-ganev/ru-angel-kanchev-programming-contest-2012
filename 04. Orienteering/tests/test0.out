05 04 01 
02 03 
01 02